src/FluentValidation.Tests/IbanValidatorTests.cs

```diff
@@ -0,0 +1,94 @@
+namespace FluentValidation.Tests
+{
+    using System.Globalization;
+    using System.Linq;
+    using System.Threading;
+    using Xunit;
+
+
+    public class IbanValidatorTests
+    {
+        TestValidator validator;
+
+        public IbanValidatorTests()
+        {
+            CultureScope.SetDefaultCulture();
+
+            validator = new TestValidator {
+                v => v.RuleFor(x => x.IbanAccount).IsIBAN()
+            };
+        }
+
+        [Fact]
+        public void IsValidTests()
+        {
+            // Empty values
+            validator.Validate(new Person { IbanAccount = null }).IsValid.ShouldBeTrue(); // Optional values are always valid
+            validator.Validate(new Person { IbanAccount = string.Empty }).IsValid.ShouldBeTrue(); // empty string
+            validator.Validate(new Person { IbanAccount = "\n\r" }).IsValid.ShouldBeTrue(); // new line

```

Comment:
What's the use case for empty string and \n\r (which is normally CRLF = \r\n) being considered valid?